src/loki/values/values.yaml

```diff
@@ -50,6 +60,12 @@ test:
   enabled: false
 # Configuration for the write pod(s)
 write:
+  # Remove default anti-affinity since this is a 1-replica setup
+  affinity: ""
```

Comment:
If affinity is an issue we might just want to do multi-node k3d in our demo bundle...I think between this and the metrics-server issue there's validation that its worth testing that way, assuming it doesn't have a massive impact on resource usage or startup time.

Reply:
I would be surprised if that doesnt result in potentially having to go to the next size runner. If that is the case maybe a better middle ground solution would be to run this (multinode) test on nightly and/or tag/release pipeline before creating release.

Reply:
@mjnagel This comment plus the one about HA leaves me thinking you'd like to see a default values.yaml for uds-core loki that would include:

* the default affinity which prevents multiple replicas from being scheduled on the same node
* replicas > 1  per component (read/write/backend) (default is 3)

I believe this would mean the default configuration would no longer work on a single node uds-k3d cluster.  Are we ok with that? I think it would mean folks would have to apply a dev config via a uds bundle config if they wanted to continue testing single node.  This would also affect us in CI for the tasks that previously were not wrapped in bundles.

Reply:
I agree the default config should be the real/ha config. The overrides being only at bundle level right now vs zarf is a little frustrating, but imo overriding the default for CI/local dev with the overrides I think makes more sense. Understood on the impact to bundle-less tasks 😢

Reply:
Maybe bundle level component selection could help work around that though  (for single package tests) 😉

Reply:
Yeah in agreement with the statements re:HA/prod config being the default. I think we definitely need to be testing the prod-ready setup with the full stack (not just single capability), so we have a few considerations here:
- single capability: this probably "has to" be the prod ready config if we want to avoid bundles for single capability, which would mean in this case it likely needs to be multi node cluster 🤔. We want this test pretty much exclusively for fast feedback, so if multi-node slows this down we'll have to rethink that.
- full stack lightweight dev: we could override with bundle overrides to make this a lightweight setup rather since we have bundle context here
- full stack prod ready: I kind of like the nightly idea here for testing, not sure if that's still in k3d vs rke2...if we want to really test prod setup k3d makes less sense. We also should balance testing in CI vs testing in our own long-lived staging, both of which are good.

This is a problem we're likely to have with other pieces of the puzzle - it's a balancing act of having uds-core in a "smart" setup out of the box but also providing the simplified dev experience. We prioritized dev experience first which I think is the right move but we need to figure out how to provide and test the prod side.

Reply:
I think I agree with everything you said here. What wasn't clicking before is that we could look at running the multi node setup just on the loki single package test.